language/en/info_acp_webpushnotifications.php

```diff
@@ -38,8 +38,17 @@
 //
 
 $lang = array_merge($lang, [
-	'ACP_WEBPUSH_SETTINGS'		=> 'Web Push settings',
-	'LOG_CONFIG_WEBPUSH'		=> '<strong>Altered Web Push settings</strong>',
+	'ACP_WEBPUSH_SETTINGS'			=> 'Web Push settings',
```

Comment:
Only thing about this, is info_acp language files are automatically loaded on every single page in the ACP, so this file should only contain lang vars we need accessible everywhere in the ACP, like LOG vars and other global lang vars.

The Lang vars that will only be used in our module should normally be kept inside their own lang file so they are only used when needed.